xdsl/dialects/llvm.py

```diff
@@ -1735,56 +1737,139 @@ def __init__(self, val: Operation | SSAValue, res_type: Attribute):
         )
 
 
+floatingPointLike = AnyOf([Float16Type, Float32Type, Float64Type])
```

Comment:
```suggestion
LLVMFloatConstr = AnyOf([Float16Type, Float32Type, Float64Type])
```

Reply:
More in line with naming for other constraints

Reply:
Note the mlir op allows vectors of floats it seems

Reply:
Switched to `AnyFloatConstr`